### PR TITLE
Object_trt documented + other shopkeeper cleanup

### PR DESCRIPTION
**Pull request by hensldm**
Documents object_trt (Kotake) as well as some small clean up around other shopkeepers.

### CODE REVIEW

**Review by tom-overton — APPROVED** [review]
Great work!

File: assets/xml/objects/object_trt.xml [hunk 1]
```diff
@@ -1,94 +1,106 @@
...
+        <!-- Animations -->
+        <Animation Name="gKotakeHoverAnim" Offset="0xA44" />
+        <Animation Name="gKotakeFlyLookAroundAnim" Offset="0x1EF4" />
+        <Animation Name="gKotakeFlyDownAnim" Offset="0x2224" />
+        <Animation Name="gKotakeFlyAnim" Offset="0x2CB0" />
+        <Animation Name="gKotakeWakeUpAnim" Offset="0x30EC" />
+        <Animation Name="gKotakeSurprisedAnim" Offset="0x3D78" />
+
+        <!-- DLists -->
+        <DList Name="gKotakePelivsDL" Offset="0x6380" />
```
Comment (engineer124) [inline assets/xml/objects/object_trt.xml:13]:
```suggestion
        <DList Name="gKotakePelvisDL" Offset="0x6380" />
```

File: assets/xml/objects/object_trt.xml [hunk 2]
```diff
@@ -1,94 +1,106 @@
...
+        <TextureAnimation Name="gKotakeEyesTexAnim" Offset="0xC988" />
+
+        <!-- Animations -->
+        <Animation Name="gKotakeHandsOnCounterAnim" Offset="0xD52C" />
+        <Animation Name="gKotakeIdleAnim" Offset="0xDE68" />
+        <Animation Name="gKotakeHalfAwakeAnim" Offset="0xEE98" />
+        <Animation Name="gKotakeSleepingAnim" Offset="0xFD34" />
+
+        <!-- Skeleton -->
+        <Limb Name="gKotakePelivsLimb" Type="Standard" EnumName="KOTAKE_LIMB_PELVIS" Offset="0xFD50" />
```
Comment (engineer124) [inline assets/xml/objects/object_trt.xml:78]:
```suggestion
        <Limb Name="gKotakePelvisLimb" Type="Standard" EnumName="KOTAKE_LIMB_PELVIS" Offset="0xFD50" />
```

**Review by AngheloAlf — APPROVED** [review]

File: assets/xml/objects/object_trt.xml [hunk 3]
```diff
@@ -1,94 +1,106 @@
...
+        <!-- DLists -->
+        <DList Name="gKotakePelvisDL" Offset="0x6380" />
+        <DList Name="gKotakeTorsoDL" Offset="0x6590" />
+        <DList Name="gKotakeNeckDL" Offset="0x66D8" />
+        <DList Name="gKotakeHeadDL" Offset="0x6808" />
+        <DList Name="gKotakeLeftHairPieceStartDL" Offset="0x7298" />
+        <DList Name="gKotakeLeftHairPieceEndDL" Offset="0x7398" />
+        <DList Name="gKotakeTeethDL" Offset="0x7478" />
+        <DList Name="gKotakeRightHairPieceStartDL" Offset="0x74C8" />
+        <DList Name="gKotakeRightHairPieceEndDL" Offset="0x75C8" />
```
Comment (EllipticEllipsis) [inline assets/xml/objects/object_trt.xml:21]: Possibly we can do a better description than this.

File: src/overlays/actors/ovl_En_Fsn/z_en_fsn.c [hunk 4]
```diff
@@ -1618,8 +1618,8 @@ void EnFsn_Draw(Actor* thisx, PlayState* play) {
 
     OPEN_DISPS(play->state.gfxCtx);
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_En_Fsn/z_en_fsn.c:1619]: Could you space out the OPEN/CLOSE macros like we usually do?

**Pull request merged**